content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,148 @@
+the next step.
+
+1. In ServiceNow, open the Service Catalog > Catalogs view by searching for
+   "Service Catalog" in the left-hand navigation.
+   1. Click the plus sign in the top right.
+   1. Select "Catalogs > Terraform Catalog > Title and Image" and choose a
+      location to add the Service Catalog.
+   1. Close the "Sections" dialog box by clicking the "x" in the upper right-hand
+      corner.
+
+-> **Note:** In step 1, be sure to choose "Catalogs", not "Catalog" from the
+left-hand navigation.
+
+## Configure VCS Repositories
+
+-> **ServiceNow Roles:** `admin` or `x_terraform.vcs_repositories_user`
+
+To make infrastructure available to your users, you must add one or more VCS
+repositories containing Terraform configuration to the Terraform service
+catalog. A single VCS repository can be used for multiple Service Items,
+potentially with different variable values.
+
+1. In ServiceNow, open the "Terraform > VCS Repositories" table by searching for
+   "terraform" in the left-hand navigation.
+1. Click "New" to add a VCS repository.
+    - Name: The name for this repository. This name will be visible to end
+      users, and does not have to be the same as the repository name as defined
+      by your VCS provider. Ideally it will succinctly describe the
+      infrastructure that will be provisioned by Terraform from the repository.
+    - OAuth Token ID: The OAuth Token ID that from your Terraform Enterprise
+      organization's VCS providers settings. This ID specifies which VCS
+      provider configured in Terraform Enterprise hosts the desired repository.
+    - Identifier: The VCS repository that contains the Terraform configuration
+      for this workspace template. Repository identifiers are determined by your
+      VCS provider; they typically use a format like
+      `<ORGANIZATION>/<REPO_NAME>` or `<PROJECT KEY>/<REPO_NAME>`. Azure DevOps
+      repositories use the format `<ORGANIZATION>/<PROJECT>/_git/<REPO_NAME>`.
+    - The remaining fields are optional.
+      - Branch: The branch within the repository, if different from the default
+        branch.
+      - Working Directory: The directory within the repository containing
+        Terraform configuration.
+      - Terraform Version: The version of Terraform to use. This will default to
+        the latest version of Terraform supported by your Terraform Enterprise
+        instance.
+1. Click "Submit".
+
+![Screenshot: ServiceNow New VCS Repository](./images/service-now-vcs-repository.png "Screenshot of the ServiceNow Terraform New VCS Repository page")
+
+**FIXME:** The current instructions seem to stop before I can actually provision
+infrastructure. Among other things, I don't see anything telling me how to
+connect the VPC repository to the catalog item. I tried looking at steps from
+the v1 integration page (below), but this doesn't seem quite right.
+
+**FIXME:** Related, the documentation on configuring variables and variable sets
```

Comment:
@pwelch ?

Reply:
I think I've got this sorted out, but would still like some more eyes on it!